views/new.erb

```diff
@@ -0,0 +1,4 @@
+<form action="/messages" method="post">
+<input type="test" name="url" />
```

Comment:
Would you consider renaming the name parameter with 'new_message' or 'new_post' instead of 'url'?